ripozo_sqlalchemy/alchemymanager.py

```diff
@@ -182,8 +182,13 @@ def retrieve_list(self, session, filters, *args, **kwargs):
         :rtype: list, dict
         """
         query = self.queryset(session)
-        pagination_count = filters.pop(self.pagination_count_query_arg, self.paginate_by)
-        pagination_pk = filters.pop(self.pagination_pk_query_arg, 1)
+        translator = _COLUMN_FIELD_MAP[int]('tmp')
```

Comment:
I'd rather we use the IntegerField directly.

    translator = IntegerField('tmp')